independent-work/bmoran/bmoran-workflow.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build and run Dockerfile for bmoran
+
+on:
+    pull_request:
+        branches:
+          - master
+        path:
+          - 'individual-work/bmoran.**'
```

Comment:
`'individual-work/bmoran/**'`

Reply:
Also please correct this file location to be in `. github/workflows/bmoran-workflow.yaml`